common/src/main/res/drawable/ic_red_circle.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="14dp"
+    android:viewportHeight="14" android:viewportWidth="14"
+    android:width="14dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#E85C4A" android:pathData="M6.9956,14C7.9574,14 8.8598,13.8183 9.7028,13.455C10.5459,13.0916 11.2898,12.5864 11.9346,11.9392C12.5794,11.292 13.0848,10.5466 13.4509,9.7031C13.817,8.8595 14,7.9585 14,7C14,6.0415 13.817,5.1405 13.4509,4.2969C13.0848,3.4534 12.5784,2.708 11.9317,2.0608C11.2849,1.4137 10.54,0.9084 9.697,0.545C8.8539,0.1817 7.9515,0 6.9897,0C6.0338,0 5.1338,0.1817 4.2898,0.545C3.4458,0.9084 2.7028,1.4137 2.061,2.0608C1.4191,2.708 0.9152,3.4534 0.5491,4.2969C0.183,5.1405 0,6.0415 0,7C0,7.9585 0.183,8.8595 0.5491,9.7031C0.9152,10.5466 1.4201,11.292 2.0639,11.9392C2.7078,12.5864 3.4517,13.0916 4.2957,13.455C5.1397,13.8183 6.0397,14 6.9956,14ZM6.9956,12.9689C6.1712,12.9689 5.3988,12.8152 4.6785,12.5078C3.9581,12.2004 3.3256,11.7732 2.7809,11.2262C2.2362,10.6792 1.8097,10.0443 1.5016,9.3216C1.1934,8.5988 1.0393,7.8249 1.0393,7C1.0393,6.1751 1.1924,5.4012 1.4986,4.6785C1.8048,3.9557 2.2303,3.3208 2.775,2.7738C3.3197,2.2268 3.9522,1.7996 4.6726,1.4922C5.3929,1.1848 6.1653,1.0311 6.9897,1.0311C7.818,1.0311 8.5929,1.1848 9.3142,1.4922C10.0355,1.7996 10.669,2.2268 11.2147,2.7738C11.7604,3.3208 12.1883,3.9557 12.4984,4.6785C12.8085,5.4012 12.9636,6.1751 12.9636,7C12.9636,7.8249 12.81,8.5988 12.5028,9.3216C12.1956,10.0443 11.7697,10.6792 11.225,11.2262C10.6803,11.7732 10.0468,12.2004 9.3245,12.5078C8.6022,12.8152 7.8259,12.9689 6.9956,12.9689ZM6.9956,8.2462C7.3057,8.2462 7.4627,8.0793 7.4667,7.7454L7.5579,4.0686C7.5619,3.9095 7.5094,3.7769 7.4004,3.6709C7.2915,3.5648 7.1546,3.5118 6.9897,3.5118C6.8189,3.5118 6.681,3.5638 6.576,3.6679C6.471,3.772 6.4205,3.9036 6.4244,4.0627L6.5039,7.7454C6.5137,8.0793 6.6776,8.2462 6.9956,8.2462ZM6.9956,10.444C7.1801,10.444 7.3411,10.3787 7.4784,10.2481C7.6158,10.1175 7.6845,9.9589 7.6845,9.7723C7.6845,9.5818 7.6168,9.4227 7.4814,9.295C7.346,9.1674 7.184,9.1035 6.9956,9.1035C6.8013,9.1035 6.6369,9.1684 6.5024,9.298C6.368,9.4276 6.3007,9.5857 6.3007,9.7723C6.3007,9.955 6.3694,10.1126 6.5068,10.2452C6.6442,10.3777 6.8071,10.444 6.9956,10.444Z"/>
+</vector>
```

Comment:
This isn't just a circle but an exclamation mark, so this file might use a rename.